src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -185,9 +189,10 @@ const cy: typeof en = ({ isDivorce, userCase, partner, isApplicant2 }: CommonCon
   awaitingFinalOrder: {
     line1: `Gallwch nawr wneud cais am 'orchymyn terfynol'. Gorchymyn terfynol yw'r ddogfen a fydd yn dod â'ch ${
       isDivorce ? 'priodas' : 'partneriaeth sifil'
-    } i ben yn gyfreithiol. Dyma'r cam olaf yn y ${
-      isDivorce ? 'broses ysgaru' : "broses i ddod â'ch partneriaeth sifil i ben"
-    }.`,
+    } i ben yn gyfreithiol.
+    Dyma'r cam olaf yn y ${isDivorce ? 'broses ysgaru' : "broses i ddod â'ch partneriaeth sifil i ben"}.`,
+    buttonText: 'Apply for a final order',
```

Comment:
```suggestion
    buttonText: 'Gwneud cais am orchymyn terfynol',
```

I found this translation by using find in files with 'Apply for a final order'